index.html

```diff
@@ -30,14 +31,100 @@ <h2>I am a Software Developer</h2>
+        <section class="main_middle_section">
+            <h3>My Recent Works</h3>
+
+            <div class="title_line"></div>
+
+            <div class="main_articles_container">
+                <article class="multi_post_card">
+                    <img class="multi_post_image" src="./assets/multi_post_image.svg" alt="stories image">
+                    <h4>Multi-Post Stories</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="techs_used">
+                        <p>css</p>
+                        <p>html</p>
+                        <p>bootsrap</p>
+                        <p>Ruby</p>
+                    </div>
+                    <button>See Project</button>
+                </article>
+
+                <article class="project_card">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="techs_used_project">
+                        <p>html</p>
+                        <p>bootsrap</p>
+                        <p>Ruby</p>
+                    </div>
+                    <button>See Project</button>
+                </article>
+
+                <article class="project_card">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="techs_used_project">
+                        <p>html</p>
+                        <p>bootsrap</p>
+                        <p>Ruby</p>
+                    </div>
+                    <button>See Project</button>
+                </article>
+
+                <article class="project_card">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="techs_used_project">
+                        <p>html</p>
+                        <p>bootsrap</p>
+                        <p>Ruby</p>
+                    </div>
+                    <button>See Project</button>
+                </article>
+            </div>
+        </section>
+
+        <section class="about_me_section">
+            <h4>About me</h4>
+            <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <button>Get My Resume</button>
+            <div class="title_line_two"></div>
+        </section>
```

Comment:
- [ ] Kindly remove the second horizontal line at the bottom of this section. Kindly compare the images attached below to see the difference between your design and that of the Figma template that you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/205187207-b100131f-aca1-479d-bf3d-1a0d2441cd75.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/205187111-95d3d2c8-aa21-491f-b16b-9909c6c2b18a.png)